src/main/java/org/synyx/urlaubsverwaltung/statistics/vacationoverview/web/ApplicationForLeaveVacationOverviewViewController.java

```diff
@@ -47,25 +79,217 @@ public String applicationForLeaveVacationOverview() {
 
     @PreAuthorize(IS_PRIVILEGED_USER)
     @GetMapping("/vacationoverview")
-    public String applicationForLeaveVacationOverview(Model model) {
-        Person signedInUser = personService.getSignedInUser();
+    public String applicationForLeaveVacationOverview(
+        @RequestParam(required = false) Integer year,
+        @RequestParam(required = false) String month,
+        @RequestParam(required = false) String department,
+        Model model, Locale locale)
+    {
+        final var signedInUser = personService.getSignedInUser();
```

Comment:
`var` empfinde ich in vielen Fällen als schwierig zu lesen. Sind wir uns einig, dass wir auf vars setzen wollen?